ansible/promtail_log_forward.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Install Promtail log forwarder
+  hosts: all
+  become: yes
+  vars:
+    ansible_ssh_common_args: '-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null'
```

Comment:
We're not sending anything sensitive to this host so it should be fine but... ugh. There has to be a better way.

(not something to fix now, I'm mostly just curious)

Reply:
Yeah. Those hosts are short lived relatively speaking and there is a right way to do this. This repo has a pre-existing use of disabling [strict key checking](https://github.com/search?q=repo:filecoin-project/lotus-infra%20StrictHostKeyChecking&type=code). Which needs a bit of work to clean up.